android/app/src/main/java/org/jitsi/meet/MainActivity.java

```diff
@@ -1,46 +1,98 @@
 package org.jitsi.meet;
 
-import com.facebook.react.ReactActivity;
-import com.facebook.react.ReactActivityDelegate;
-import com.facebook.react.ReactRootView;
-
-public class MainActivity extends ReactActivity {
-    /**
-     * {@inheritDoc}
-     *
-     * Overrides {@link ReactActivity#createRootActivityDelegate()} to customize
-     * the {@link ReactRootView} with a background color that is in accord with
-     * the JavaScript and iOS parts of the application and causes less perceived
-     * visual flicker than the default background color.
-     */
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Build;
+import android.provider.Settings;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
+import org.jitsi.jitsikit.*;
+
+
+/**
+ * The one and only Activity that Jitsi Meet (the app) needs. The activity is launched in
```

Comment:
Sure. I didn't pay much attention to that when doing the view part, I'll fix those uop ASAP.